.github/workflows/spelling.yml

```
@@ -0,0 +1,11 @@
+name: Spell checking
+on: [push, pull_request]
+
+jobs:
+  spelling:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - uses: check-spelling/check-spelling@main
```

```suggestion
    - uses: check-spelling/check-spelling@v0
```
I think it's better to pin to the latest major release, this tool is still under active development so things will break. Looks like development goes in the [`prerelease`](https://github.com/check-spelling/check-spelling/tree/prerelease) and `main` branch actually has the latest release but we never know if it changes.
And it's a good practice to pin to release.

---

Alright. Will keep in mind. Thanks!